libs/dal/src/repositories/layout/layout.repository.ts

```diff
@@ -66,27 +66,27 @@ export class LayoutRepository extends BaseRepository<LayoutDBModel, LayoutEntity
     return await this.findOne({ _environmentId, _organizationId, isDefault: true });
   }
 
-  async findDeleted(id: LayoutId, environmentId: EnvironmentId): Promise<LayoutEntity | undefined> {
+  async findDeleted(id: LayoutId, environmentId: EnvironmentId): Promise<LayoutEntity | null> {
```

Comment:
What's the reason behind moving from undefined returns to null? Just asking as I deliberately chose in this implementation (and I would like to do so in general) to go for `undefined` rather than use `null` for the type implications (being `null` an `Object` I'd rather avoid to use it as much as possible; e.g. default values in arguments when passing `null` wouldn't be used).

Reply:
The types that are returned by the find model methods are like `Entity | null`. And line 75 does return null too.